app/src/main/java/br/com/fundatec/fundatecheros/home/view/SplashActivity.kt

```diff
@@ -0,0 +1,29 @@
+package br.com.fundatec.fundatecheros.home.view
+
+import android.content.Intent
+import android.os.Bundle
+import android.os.Handler
+import androidx.appcompat.app.AppCompatActivity
+import br.com.fundatec.fundatecheros.R
+
+class SplashActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_splash)
+
+        var handler = Handler();
+        handler.postDelayed(Runnable() {
+            run() {
+                time()
+            }
+        }, 3000);
```

Comment:
Para remover o deprecated, nós podemos fazer dessa forma
```suggestion
        Handler(Looper.getMainLooper()).postDelayed({
            run() {
                time()
            }
        }, 3000)
```